examples/notebooks/Untitled.ipynb

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
Was it intended to add this notebook called "Untitled.ipynb"?

Reply:
Removed